models/api-user.test.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const ApiUser = require('./api-user')
+const factory = require('../test/factory')
+
+describe('ApiUser', () => {
+  it('should be defined', () => {
```

Comment:
I think most of the project is moving to `test` as compared to `it`. Not sure, but it seems like any new tests should follow suite.